nixos/modules/programs/browserpass.nix

```diff
@@ -6,21 +6,17 @@ with lib;
 
   ###### interface
   options = {
-    programs.browserpass = {
-      enable = mkOption {
-        default = false;
-        type = types.bool;
-        description = ''
-          Whether to install the NativeMessaging configuration for installed browsers.
-        '';
-      };
-    };
+    programs.browserpass.enable = mkEnableOption "the NativeMessaging configuration for installed browsers.";
```

Comment:
If I understand correctly that would be implemented in the firefox [wrapper](https://github.com/NixOS/nixpkgs/blob/700d6186a30e9e8118a6b319f12cb5cfb9d086bd/pkgs/applications/networking/browsers/firefox/wrapper.nix). Let's merge this first and open another issue for firefox support if desired.